README.md

```diff
@@ -65,7 +65,7 @@ Cell is a struct that represents one cell on the terminal. We can set its backgr
 Let's make a nice pretty lake, too. We'll use a [Rectangle](http://godoc.org/github.com/JoelOtter/termloop#Rectangle) for this. We'll put the lake at position (10, 10), with width 50 and height 10. All measurements are in terminal characters! The last argument is the colour of the Rectangle.
 
 ```go
-level.AddEntity(tl.NewRectangle(10, 10, 50, 20, tl.ColorBlue))
+level.AddEntity(tl.NewRectangle(10, 10, 50, 10, tl.ColorBlue))
```

Comment:
Could you set this back to 20, and then update the description above to use 20 instead of 10 for the height? Thanks!